### DISCUSSION TITLE
Combination of stacker and volume_mesh

### DISCUSSION THREAD

**Discussion by aglowinski-danfoss** [ansys/pyprimemesh#697] [Q&A]
Your program is working great for our pcbs with this [Example](https://prime.docs.pyansys.com/version/dev/examples/gallery_examples/gallery/09_multi_layer_quad_mesh_pcb.html#sphx-glr-examples-gallery-examples-gallery-09-multi-layer-quad-mesh-pcb-py)! For our simulations we want to drive the meshing further and combine this stacker method with a volume mesh as shown in this [Example](https://prime.docs.pyansys.com/version/dev/examples/gallery_examples/gallery/07_saddle_bracket.html#sphx-glr-examples-gallery-examples-gallery-07-saddle-bracket-py).
The input CAD geometry contains both the pcb and the "brackets" on top of the pcb, so the idea is to mesh the pcb first (and part of the surface of the bracket) and the bracket afterwards.
Is this possible? 
I am looking forward for your answer and thank you in advance!

**Comment by waltersma** [comment]
Hi,

As of today this is not possible but I will convert this to an enhancement and then we can provide you updates on this.

Thanks

**Reply by aglowinski-danfoss** [reply]
Hi,

great! That would be a massive improvement for our work. If needed, I can provide you a sample spaceclaim geometry.

Best regards,

**Reply by aglowinski-danfoss** [reply]
Hello,
I just saw, that from 0.5 on, Prime contains Multizone Meshing. Can this feature be used to solve this problem?
Best regards,
